core/controlplane/config/templates/cloud-config-controller

```diff
@@ -791,6 +791,116 @@ write_files:
       sed -i -e "s#\$ETCDKEY#$etcd_key#g" /srv/kubernetes/manifests/calico.yaml
 
 {{ end }}
+{{ if .ExportKubeResources }}
+  - path: /srv/kubernetes/manifests/cluster-dump.yaml
```

Comment:
If we go [that way](https://github.com/kubernetes-incubator/kube-aws/pull/507/commits/20a1f392a5925ccdf728e7373000b508d0c7b364#r111071194), I'd expect this to be named like `kube-resources-autosave.yaml` so that it will be consistent across configuration key names, func names, and manifest file names.